packages/@react-aria/actiongroup/src/ActionGroupKeyboardDelegate.ts

```diff
@@ -17,11 +17,13 @@ export class ActionGroupKeyboardDelegate<T> implements KeyboardDelegate {
   private collection: Collection<T>;
   private flipDirection: boolean;
   private disabledKeys: Set<Key>;
+  private isDisabled: boolean;
 
   constructor(collection: Collection<T>, direction: Direction, orientation: Orientation, disabledKeys: Set<Key> = new Set()) {
     this.collection = collection;
     this.flipDirection = direction === 'rtl' && orientation === 'horizontal';
     this.disabledKeys = disabledKeys;
+    this.isDisabled = collection.size === disabledKeys.size;
```

Comment:
Should we have `isDisabled` be part of the args passed to the delegate? Might give some more flexibility that way

Reply:
not quite sure what you're asking? this is a slightly different disabled than saying the group is disabled, this is inferring that it should be disabled
are you just asking to pass this equality check in to the delegate though?